test/src/test/java/org/corfudb/runtime/collections/CorfuStoreShimTest.java

```diff
@@ -248,7 +249,7 @@ public void checkRevisionValidation() throws Exception {
         }
         assertThat(entry.getMetadata().getRevision()).isEqualTo(2L);
         assertThat(entry.getMetadata().getCreateUser()).isEqualTo("user_1");
-        assertThat(entry.getMetadata().getLastModifiedTime()).isLessThan(System.currentTimeMillis());
+        assertThat(entry.getMetadata().getLastModifiedTime()).isLessThan(System.currentTimeMillis() + 1);
```

Comment:
Maybe consider using `.isLessThanOrEqualTo()` instead?

Reply:
fone

Reply:
@xnull The change to isLessThanOrEqualTo() did not take effect in the patch